server/app/views/applicant/ApplicantCommonIntakeUpsellTemplate.html

```diff
@@ -0,0 +1,57 @@
+<html th:attr="lang=${preferredLanguage.code()}">
+  <head
+    th:replace="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}"
+  ></head>
+  <body data-load-modal="true">
+    <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
+    <main role="main">
+      <h1
+        th:text="#{title.commonIntakeConfirmationTi}"
+        th:if="${isTrustedIntermediary}"
+      ></h1>
+      <h1
+        th:text="#{title.commonIntakeConfirmation}"
+        th:if="${!isTrustedIntermediary}"
+      ></h1>
+
+      <div th:if="${eligiblePrograms.size()==0}" style="margin-bottom: 20px">
+        <p
+          th:utext="#{content.commonIntakeNoMatchingProgramsTi(${moreResourcesLinkHtml})}"
+          th:if="${isTrustedIntermediary}"
+        ></p>
+        <p
+          th:utext="#{content.commonIntakeNoMatchingPrograms(${moreResourcesLinkHtml})}"
+          th:if="${!isTrustedIntermediary}"
+        ></p>
+      </div>
+
+      <div th:if="${eligiblePrograms.size()>0}" style="margin-bottom: 20px">
+        <p
+          th:text="#{content.commonIntakeConfirmationTi}"
+          th:if="${isTrustedIntermediary}"
+        ></p>
+        <p
+          th:text="#{content.commonIntakeConfirmation}"
+          th:if="${!isTrustedIntermediary}"
+        ></p>
+
+        <div th:each="ep : ${eligiblePrograms}">
+          <h2 th:text="${ep.getTitle()}">Program Name</h2>
+          <p th:text="${ep.getDescription()}">Program Description</p>
+        </div>
+      </div>
+
+      <a
+        href="/"
+        class="usa-button"
+        id="another-program"
+        th:text="#{link.applyToAnotherProgram}"
+      >
+      </a>
+    </main>
+
```

Comment:
add the modal container fragment here

Reply:
Is clicking DevTools supposed to show the modal fragment? (It doesn't. Do I need to do something extra?)